closure/goog/base.js

```diff
@@ -1396,6 +1396,7 @@ goog.loadFileSync_ = function(src) {
   if (goog.global.CLOSURE_LOAD_FILE_SYNC) {
     return goog.global.CLOSURE_LOAD_FILE_SYNC(src);
   } else {
+    /** @type {XMLHttpRequest} */
     var xhr = new goog.global['XMLHttpRequest']();
```

Comment:
The other option would be to increase the bracket usage:

    xhr['open']('get', src, false);

This is also occurring on [Line 850](https://github.com/google/closure-library/blob/master/closure/goog/base.js#L850) which doesn't use brackets.